jsapp/js/components/modalForms/projectSettings.es6

```diff
@@ -507,12 +553,10 @@ class ProjectSettings extends React.Component {
             </button>
           }
 
-          {this.props.context !== PROJECT_SETTINGS_CONTEXTS.REPLACE &&
-            <button onClick={this.displayStep.bind(this, this.STEPS.CHOOSE_TEMPLATE)}>
-              <i className='k-icon-template' />
-              {t('Use a template')}
-            </button>
-          }
+          <button onClick={this.displayStep.bind(this, this.STEPS.CHOOSE_TEMPLATE)}>
+            <i className='k-icon-template' />
+            {t('Use a template')}
```

Comment:
I don't think this should appear before the XLSForm options when replacing a project, since those are likely to be used far more frequently.

Reply:
@jnm would it be ok if template option would be last when replacing project and also when creating project?

Reply:
@jnm I am displaying the template as last for replace project, but as second for new project :)